package.json

```diff
@@ -31,7 +31,7 @@
     "react-markdown": "^8.0.7",
     "react-vanilla-tilt": "^1.0.0",
     "sass": "^1.45.1",
-    "styled-components": "^5.3.6",
+    "styled-components": "^5.3.11",
```

Comment:
Any specific reason you'd want to upgrade the version ?

Reply:
nh i just used in my case

Reply:
You should avoid upgrading version if not required for this npm ecosystem. Reason is that if you upgrade the version, We would also need to lock the newer version in lock files.  And this requires you to also commit the lock-file. Now we can have a new lock file but then it brings conflicts to other contributors who are working and makes their task a bit problematic and they need to spend time in resolving version conflicts. But if we need a strict requirement for such version upgrade then it has to be prioritize and contributors have to deal with the problem. But unless and until it is important and prioritize,  We always try to avoid that.